### ISSUE TITLE
Ability to disable the formatter for a block of SQL

### ISSUE THREAD

**Issue by mbtolou** [sql-formatter-org/sql-formatter#703]
how to ignore formatter for block of sql

like this :
```

// prettier-ignore
or
-- prettier-ignore

```

**Comment by nene** [comment]
SQL Formatter is not Prettier. It has not been developed to be compatible with Prettier. Heck, the first version of SQL Formatter was released before Prettier existed.

I know there exists [prettier-plugin-sql](https://github.com/un-ts/prettier/tree/master/packages/sql), which uses SQL Formatter under the hood, but I have no control over that project.

If you're interested in more Prettier-compatible SQL formatting solution, I suggest you try [prettier-plugin-sql-cst](https://github.com/nene/prettier-plugin-sql-cst). This doesn't support the ignore-comments, but there exists more of a chance that such a feature gets implemented.

In this project, the best one could hope for would be a feature where you mark both start and end of the formatted code, like:

```sql
-- sql-formatter:ignore-start
SELECT * FROM tbl
-- sql-formatter:ignore-end
```

There could be a point for implementing something like this, as it would help in working around scenarios where the formatter currently fails to produce a sensible output or even crashes. Like:

- #184
- #144

**Comment by nene** [comment]
Implemented in 15.1.0 release.
